lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -288,6 +288,18 @@ public function getDateTimeTypeDeclarationSQL(array $fieldDeclaration)
     /**
      * {@inheritDoc}
      */
+    public function getTimestampTypeDeclarationSQL(array $fieldDeclaration)
+    {
+        if (isset($fieldDeclaration['notnull']) && $fieldDeclaration['notnull'] == true) {
+            return 'TIMESTAMP';
+        }
+
+        return 'TIMESTAMP NULL';
```

Comment:
This does not seems to be the best way to make it nullable, but was unsure where else to put this. I'm open for ideas.